requirements/base.txt

```diff
@@ -38,7 +40,7 @@ gitpython>=3.1.37; sys_platform == 'win32'
 lxml>=4.6.3; sys_platform == 'win32'
 pymssql>=2.2.1; sys_platform == 'win32'
 pymysql>=1.0.2; sys_platform == 'win32'
-pythonnet>=3.0.1; sys_platform == 'win32'
+pythonnet>=3.0.0; sys_platform == 'win32'
```

Comment:
It's interesting why this works:
https://pypi.org/project/pythonnet/#history

Reply:
Yeah, we shouldn't proceed like this. I'll add a constraint for 3.13.